packages/signature-controller/src/SignatureController.test.ts

```diff
@@ -426,15 +474,11 @@ describe('SignatureController', () => {
 
         expect(keyringControllerMethod).toHaveBeenCalledTimes(1);
         expect(keyringControllerMethod).toHaveBeenCalledWith(
-          messageParamsMock2,
+          messageParamsMock3,
```

Comment:
Would using a spread operator here make it more explicit what the key input to the test is?

Something like `{...messageParamsMock3, deferSetAsSigned: true}`?

Reply:
Good idea - this allowed me to remove the third mock